src/storage/src/hummock/sstable/builder.rs

```diff
@@ -121,17 +150,40 @@ impl SstableBuilder {
 
         // TODO: refine me
         value.encode(&mut self.raw_value);
+        let mut extract_key = user_key(full_key);
         if let Some(table_id) = get_table_id(full_key) {
             if self.last_table_id != table_id {
                 self.table_ids.insert(table_id);
                 self.last_table_id = table_id;
             }
+
+            extract_key = match &mut self.filter_key_extractor {
```

Comment:
IMO, its a mapping between `filter_key_extractor` and `table_id`, we can only found the `filter_key_extractor` of `table_schema` from `table_id`.  What is the ideal implementation?

Reply:
disscuss with wallace, filter_key_extractor may be another type like FixedLengthFilterKeyExtractor, so its not a bind between table_id and FilterKeyExtractor. I remove table_id check, and call extract directly.

and remove the length check for MultiFilterKeyExtractor, when length not match, it will return the full_key